tests/unit/AmalgamationSelection.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('AmalgamationSelection', () => {
 
   beforeEach(() => {
     const router = mockRouter.mock()
-    router.push({ name: 'amalgamation-selection', params: { filingId: '0' } })
```

Comment:
The page doesn't verify that this route parameter exists, so we don't need to set it here.